Jenkinsfile

```diff
@@ -171,7 +171,10 @@ pipeline {
       }
     }
     stage('Intel Device plugins') {
-      when { changeRequest() }
+      when {
+	beforeAgent true
+	allOf { changeRequest(); environment name: 'BMAAS', value: 'yes' }
```

Comment:
nit: I'm not Jenkins expert, but I'd expect indentation here.

Reply:
interesting. It shows OK with my `$EDITOR`